### DISCUSSION TITLE
Taking a video in AR WebXR (question)?

### DISCUSSION THREAD

**Discussion by Dovatech** [google/model-viewer#1947] [Q&A]
Hello, 
Is it possible to take a video in AR **WebXR** ? I know that it is possible in Scene Viewer and Quick look.  Can I do the same in WebXR?
Thank you)

### Browser Affected

- [x ] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [x ] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

**Comment by elalish** [comment]
Not yet, but we're working on it.
